src/wordCount/Main.java

```diff
@@ -0,0 +1,63 @@
+
+        // System.out.println(text);
+
+        String text = text2.toLowerCase();
```

Comment:
You don't need to create a second text variable. You can just assign a modified value of the `text` variable, like so:
```
text = text.toLowerCase();
```